README.md

```diff
@@ -95,19 +95,17 @@ Visit [Rule examples](https://semgrep.dev/docs/writing-rules/rule-ideas/) for us
 
 ### Integrations
 
-Visit [Integrations](https://semgrep.dev/docs/integrations/) to learn about Semgrep editor, commit, and CI integrations.
+Visit [Integrations](https://semgrep.dev/docs/integrations/) to learn about Semgrep editor, commit, and CI integrations. When integrated into CI, Semgrep will automatically only show findings that are being introduced in the current commit.
 
-### FAQ
 
-Visit [FAQ](https://semgrep.dev/docs/faq/) to learn more. Semgrep is supported by [r2c](https://r2c.dev).
+### More
 
-### Contributing
+* [Frequently asked questions (FAQs)](https://semgrep.dev/docs/faq/)
+* [Contributing](https://semgrep.dev/docs/contributing/)
+* [Ask questions in the r2c Community Slack](https://r2c.dev/slack)
+* [CLI usage and exit codes](https://semgrep.dev/docs/cli-usage)
+* [r2c Youtube channel with Semgrep presentation videos](https://www.youtube.com/channel/UC5ahcFBorwzUTqPipFhjkWg)
 
-Visit [Contributing](https://semgrep.dev/docs/contributing/) or join the [r2c Community Slack](https://r2c.dev/slack) to learn more. Semgrep is LGPL-2.1 licensed. 
```

Comment:
Why omit the mention of licensing? Is it expected that people would go and just read the `LICENSE` file?

Reply:
My thought was that it's redundant, but on reflection I suppose it doesn't hurt to have it in the readme either!